bw2data/backends/proxies.py

```diff
@@ -477,6 +479,8 @@ def new_edge(self, **kwargs):
         exc = Exchange()
         exc.output = self.key
         for key in kwargs:
+            if key == "id":
+                raise ValueError("`id` must be created automatically")
```

Comment:
Do we need the same validation for nodes? https://github.com/brightway-lca/brightway2-data/blob/b716b1420a1874cdaf139fae57494ec54beac994/bw2data/backends/base.py#L680

Reply:
Fixed in https://github.com/brightway-lca/brightway2-data/pull/202/commits/623f44135426a10d394fcd9834f7f35b428c660d